esmvaltool/preprocessor/_regrid.py

```diff
@@ -194,7 +194,13 @@ def regrid(src_cube, target_grid, scheme):
                 src_cube.remove_coord(coord)
 
     # Perform the horizontal regridding.
-    result = src_cube.regrid(target_grid, horizontal_schemes[scheme])
+    try:
```

Comment:
Is this really needed? Input cubes are already logged by the preprocessor runner and the information is available in the file run/main_log_debug.txt.

Reply:
Yes, definitely. Yesterday I tried to find the model which failed at regridding along ~100 other models. The `main_log_debug` contains 72000 lines with the output of all processes (in my case 8). So the cube which is printed right before the error message has (most of the time) nothing to do with the actual error, there were even cubes printed **after** the traceback. Searching for the `pid` does not help either -- the error is printed by another one than the debug output.

Reply:
Still, I think this is something that should be implemented in the preprocessor runner, not in preprocessor functions. If it is needed for this one, it will also be needed for other functions.

Reply:
I'm rewriting the runner at the moment to support provenance https://github.com/ESMValGroup/ESMValTool/issues/240, so let's leave it for now and I will add it in there.

Reply:
Ok, sure, that's even better :+1: . I will adapt the PR.